python/ml4ir/base/config/parse_args.py

```diff
@@ -83,6 +83,13 @@ def define_args(self):
             help="Path to YAML file or YAML string with feature metadata for training.",
         )
 
+        self.add_argument(
+            "--evaluation_config",
+            type=str,
+            help="Path to YAML file or YAML string with parameters needed for model evaluation of the test set.",
+            default="ml4ir/base/config/default_evaluation_config.yaml"
```

Comment:
@mohazahran Would this be messing up things for `ClassificationModel`?

Reply:
nope. I handled that